### ISSUE TITLE
Option to display committer date in commit list

### ISSUE THREAD

**Issue by abentele** [ForkIssues/TrackerWin#144]
In the view "All commits", the displayed date is sometimes not as expected:

If, for example, the commit date is "21 Jul 2017", and I've cherry-picked the commit on "01 Jan 2019", then I expect this view shows "01 Jan 2019". But it shows "21 Jul 2017".

I think it's not wrong, but the date when I've cherry-picked the commit is the information which is more useful than the date of the original commit.

Atlassian Sourcetree would show "01 Jan 2019".
git log would show "21 Jul 2017".

(the commit view shows both dates, so the information can be viewed, but not in the overview table).

**Comment by DanPristupov** [comment]
There are two dates:
- Author date. When the changes were made
- Committer date. When commit (SHA) was created.

Generally speaking, committer date is more important. Until you use rebase/fixup/reword, which will rewrite ALL the committer dates till the commit you change.

So the author date is more reliable for 99.9% of users:
1. it represents the time the feature was impelemented.
2. it will not be rewritten by cherry-pick/rebase/amend and users won't see same dates in the commit list after rebase.

**Comment by abentele** [comment]
Why the commiter date is often more important for me in the git log: when I look on this view, it's the change log of the product I'm currently working on. And rather often my question is: what has been changed in this product yesterday (for example, because I know yesterday the software did work, while today we observed a new bug). For this question, the author date is irrelevant.

For me, it is sufficient to display the author date only in the commit detail view (as it's already implemented).

**Comment by DanPristupov** [comment]
@abentele thank you for describing your use case. It makes sense.

It would be great to have an option. However the revision processing in Fork is very optimized and the change is not as simple as it seems. So, I don't have any ETA, there are too many more important tasks right now.

**Comment by abentele** [comment]
agree, and it's not very important, and it shouldn't lead to worse performance (while the performance now is very good!).
Thanks for discussing this issue with me!

**Comment by harrydet** [comment]
Just wanted to say that I would also find this option very useful.

**Comment by fmercier** [comment]
Also BitBucket server show a special icon when the Committer is different than the Author; This information is also useful.
The mouse over on this icon display both names. This might also be applied when the  dates are different.

**Comment by harrypurba** [comment]
This would be a nice feature to have. The author's date on the list seems somewhat irrelevant. Thanks!
